src/assets/scss/_variables.scss

```diff
@@ -0,0 +1,43 @@
+$primary: #f15151;
+$error: #ef5777;
+$success: #1abc9c;
+$light: #f5f8fa;
+$medium: #657786;
+$dark: #34495e;
+$white: #fff;
+$black: #000;
+
+$offWhite: #f8f8f8;
+$medGray: #858585;
+$darkGray: #3C3C3C;
+
+$emGreen: #1AA9A5;
+$chrisGreen: #105351;
+$sangBlue: #508197;
+$einBlue: #92C3E6;
+
+$inactiveBlue: #32A0F2;
```

Comment:
can we call this yuxuanBlue 🥺🥰 so each designer gets a blue color haha, thank you!

Reply:
Updated!